blog/2024-08-20-nushell_0_97_0.md

```diff
@@ -141,7 +141,7 @@ After [#13612](https://github.com/nushell/nushell/pull/13612), `glob` now also t
 
 ### `into datetime` [[toc](#table-of-content)]
 
-Thanks to [@NotTheDr01ds](https://github.com/NotTheDr01ds) in [#13541](https://github.com/nushell/nushell/pull/13541), integer values can now be piped into `into datetime` in combination with a format string. The integers will be interpreted as nanosecond-precision UNIX timestamps.
+Thanks to [@NotTheDr01ds](https://github.com/NotTheDr01ds) in [#13541](https://github.com/nushell/nushell/pull/13541), integer values can now be piped into `into datetime` in combination with a format string. The integers will be interpreted based on the format string.
```

Comment:
Maybe worth restricting that this requires a format string accepting a single value, right?

Reply:
Yes, AFAIK it's only useful with one format string (`%s`) (but a common enough one that I felt it was worthwhile).  I'll add the example.

Reply:
Thanks!